### PR TITLE
Fix progress circle

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #876
[Problem] Progress circle was different in older version
[Solution]
 - Make progress thinner
 - Edit progress example

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Comment by korneliakobiela** [comment]
I will make another commit with changing place of progress-circle example when #886 will be merged.

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: src/css/profile/mobile/common/progress.less [hunk 1]
```diff
@@ -91,18 +91,18 @@ tau-progress {
 }
 
 .ui-progress-circle.ui-progress-circle-large {
-	width: 124 * @unit_base;
-	height: 124 * @unit_base;
+	width: 124 * @px_base;
```
Comment (singa2000) [inline src/css/profile/mobile/common/progress.less:94]: I have a question.
What I've seen the size in the Guide for Progress circle(Indeterminated)
- Large progress circle : 60*60
- Medium progress circle : 48*48
- Small progress circle : 24*24.

How do you get the width, height  value?

**Review by singa2000 — COMMENTED** [review]
Is the PR  also implemented for Progress circle(Indeterminate)?
If no, it just implement for Progress circle(Determinate)?

File: src/css/profile/mobile/common/progress.less (hunk 1 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/progress.less:94]: This is the determinate Progress Circle. It has FABs width and height which is 56*56. Probably next I will make indeterminate progress elements, so I will change these parameters

**Comment by korneliakobiela** [comment]
This PR includes only determinate Progress Circle.

**Pull request merged**